intuita-webview/src/fileExplorer/TreeView/index.tsx

```diff
@@ -77,7 +77,11 @@ const TreeView = ({
 	stagedJobs,
 }: Props) => {
 	const allFileNodesReady = fileNodes !== null;
+
+	// @TODO  @UX Need to show info message to users.
```

Comment:
@DmytroHryshyn is there a ticket for it?

Reply:
i will add a ticket